packages/html/src/grid/tests/grid-column-reordering.tsx

```diff
@@ -0,0 +1,260 @@
+import { Chip, ChipAction, ChipList } from '../../chip';
+import { Icon } from '../../icon';
+import { Pager } from '../../pager';
+import { Table, TableThead, TableTbody, TableRow, TableTh, TableTd } from '../../table';
+import { Toolbar } from '../../toolbar';
+
+const root = ReactDOM.createRoot(
+    document.getElementById('app') as HTMLElement
+);
+
+const styles = `
+    .k-animation-container,
+    .k-popup {
+        position: relative;
+        overflow: visible;
+    }
+
+    #discontinued, #discontinued .k-popup {
+        top: -114px;
+        left: 254px;
+    }
+
+    #category, #category .k-popup {
+        top: -337px;
+        left: 120px;
+    }
+`;
+
+root.render(
+    <>
+        <style>{styles}</style>
+        <div id="test-area">
+
+            <section>
+                <div className="k-grid k-grid-md k-grid-no-scrollbar">
+                    <Toolbar className="k-grid-toolbar">
+                        <ChipList>
+                            <Chip text="Units in Stock" icon="sort-asc-small" actions={
+                                <>
+                                    <ChipAction type="more"/>
+                                    <ChipAction type="remove"/>
+                                </>
+                            } />
+                            <Chip text="Category" icon="sort-desc-small" actions={
+                                <>
+                                    <ChipAction type="more"/>
+                                    <ChipAction type="remove"/>
+                                </>
+                            } />
+                        </ChipList>
+                    </Toolbar>
+                    <div className="k-grid-header">
+                        <div className="k-grid-header-wrap">
+                            <Table size="medium" className="k-grid-header-table" >
+                                <colgroup>
+                                    <col />
+                                    <col />
+                                    <col />
+                                </colgroup>
+                                <TableThead>
+                                    <TableRow>
+                                        <TableTh className="k-header k-sorted">
+                                            <span className="k-cell-inner">
+                                                <span className="k-link">
+                                                    <span className="k-column-title">Product Id</span>
+                                                    <span className="k-sort-icon"><Icon icon="sort-asc-small" /></span>
+                                                </span>
+                                            </span>
+                                            <a href="#" className="k-grid-header-menu k-grid-column-menu">
+                                                <Icon icon="more-vertical" />
+                                            </a>
```

Comment:
This rendering is not correct - the column menu toggle should be inside the `k-cell-inner` element:

```suggestion
                                            <span className="k-cell-inner">
                                                <span className="k-link">
                                                    <span className="k-column-title">Product Id</span>
                                                    <span className="k-sort-icon"><Icon icon="sort-asc-small" /></span>
                                                </span>
                                                <a href="#" className="k-grid-header-menu k-grid-column-menu">
                                                    <Icon icon="more-vertical" />
                                                </a>
                                            </span>
```